### PR TITLE
Disablers can no longer execute people

### PR DESCRIPTION
**Pull request by zzzmike**
# About the pull request

Disablers can't execute people anymore.

# Explain why it's good for the game

This doesn't really make sense to be in the game, I've never seen anyone intentionally try to execute with a disabler before, it's always some poor newbie MP who does it by accident and gets PTSD afterwards.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
del: Disablers can't execute people anymore
/:cl:

### CODE REVIEW

**Comment by iloveloopers** [comment]
Removes soul

**Comment by zzzmike** [comment]
> Removes soul

Erm actually it preserves soul (of the character getting accidentally executed) 🤓

File: code/modules/projectiles/guns/energy.dm [hunk 1]
```diff
@@ -214,6 +214,7 @@
 	charge_icon = "+taser"
 	black_market_value = 20
 	actions_types = list(/datum/action/item_action/taser/change_mode)
+	flags_gun_features = GUN_CANT_EXECUTE
```
Comment (Drulikar) [inline code/modules/projectiles/guns/energy.dm:217]: You need to retain the gun features from parent. So either

`flags_gun_features = GUN_UNUSUAL_DESIGN|GUN_CAN_POINTBLANK|GUN_CANT_EXECUTE`
or
`flags_gun_features = parent_type::flags_gun_features | GUN_CANT_EXECUTE`
But I will note its not the norm to do the latter at least yet (515 feature).

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Comment by zzzmike** [comment]
thanks, also do you know why the bot removed the stale flag? was it because I made a comment ~20 hours earlier?

**Comment by Drulikar** [comment]
> thanks, also do you know why the bot removed the stale flag? was it because I made a comment ~20 hours earlier?

Yes the stale workflow is just trying to detect inactivity

**Review by Drulikar — APPROVED** [review]

**Comment by RavinHobo** [comment]
1984

**Pull request merged**